src/v3/test/integration/authenticator-expiry-warning-password.test.tsx

```diff
@@ -30,7 +30,7 @@ describe('authenticator-expiry-warning-password', () => {
 
     const titleElement = await findByText(/Your password will expire in/);
     await waitFor(() => expect(titleElement).toHaveFocus());
-    await findByText(/When your password expires you will be locked out of your Okta account./);
+    await findByText('When your password expires, you will have to change your password before you can login to your Localhost account.');
```

Comment:
Related to this commit: https://github.com/okta/okta-signin-widget/commit/6e25d801f521577f5f321d8801eac6e37f4c1121